SmartDeviceLink/SDLLifecycleConfiguration.m

```diff
@@ -102,6 +102,7 @@ - (id)copyWithZone:(nullable NSZone *)zone {
     newConfig->_tcpDebugIPAddress = _tcpDebugIPAddress;
     newConfig->_tcpDebugPort = _tcpDebugPort;
     newConfig->_appType = _appType;
+    newConfig->_additionalAppTypes = _additionalAppTypes;
     newConfig->_language = _language;
     newConfig->_languagesSupported = _languagesSupported;
     newConfig->_appIcon = _appIcon;
```

Comment:
Shouldn't the check for `isMedia` on line 70 include the `additionalAppTypes`?
```
- (BOOL)isMedia {
    if ([self.appType isEqualToEnum:SDLAppHMITypeMedia]) {
        return YES;
    }

    return NO;
}
```

Reply:
Fixed